Source/ViewControllers/BrickCollectionView.swift

```diff
@@ -501,7 +509,10 @@ extension BrickCollectionView: AsynchronousResizableDelegate {
         weak var weakLayout = self.layout
         self.performBatchUpdates({
             weakLayout?.updateHeight(indexPath, newHeight: newHeight)
-            }, completion: completion)
+        }, completion: { completed in
+            let _ = self
```

Comment:
I was still reproducing the crash if I made this `performBatchUpdates` capture `self`. But it does seem this is where the cause of the crash is as the crash no longer happens if I add `let _ = self` into the completion handler.

I kind of think that we should keep `let _ = self` in all completion handlers to avoid the issue appearing anywhere else. What do you think Ruben?